### PR TITLE
Add base configuration for Android

### PR DESCRIPTION
**Pull request by rastislavvasko**
Adds the base Android configuration. What we already use in Todoist et al, plus some additional package rules and a weekly schedule.

Cc @jankratochvilcz for visibility. :)

### CODE REVIEW

**Review by pedrodanielcsantos — COMMENTED** [review]
Looks good! 👌 Will be great to have this shared!

File: android-base.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        "org.jetbrains.kotlin."
+      ]
+    },
+    {
+      "groupName": "org.jetbrains.kotlinx.*",
+      "matchPackagePrefixes": [
+        "org.jetbrains.kotlinx:",
+        "org.jetbrains.kotlinx."
+      ]
+    },
```
Comment (goncalossilva) [inline android-base.json:23]: Curious about this. 😊 `org.jetbrains.kotlinx` don't usually go hand-in-hand, do they?

Comment (rastislavvasko) [reply android-base.json:23]: They don't, it's just to have them in a single PR. :)

Comment (goncalossilva) [reply android-base.json:23]: Ok! I've been using this feature when updates follow logical groups, hence the confusion. 😄

Comment (rastislavvasko) [reply android-base.json:23]: Yeah it's not a strictly logical group. We were trying to cut down on the number of PRs (without going all the way to a single PR). 😅

**Pull request merged**